checkbox-provider-kivu/bin/compare_power_consumption.py

```diff
@@ -26,6 +26,17 @@ def load_energy(fname: str ) -> tuple[float, dict[str, float]]:
 
     return (duration, energy_usage)
 
+def get_energy_usage(rapl_output):
+    consumed_enegry = 0.0
+    p0 = rapl_output.get('package-0', None)
+    if p0:
+        consumed_enegry = p0
```

Comment:
aren't the 4 lines above equivalent to consumed_energy = rapl_output.get('package-0', 0.0)?